src/briefcase/platforms/macOS/__init__.py

```diff
@@ -711,16 +708,10 @@ def package_app(
                 raise BriefcaseCommandError(
                     "Can't notarize an app with an ad-hoc signing identity"
                 )
-            self.logger.info(
-                "Signing app with ad-hoc identity...",
-                prefix=app.app_name,
-            )
+            self.logger.info("Signing app with ad-hoc identity...", prefix=app.app_name)
             self.logger.warning(
-                (
-                    "Because you are signing with the ad-hoc identity, this "
-                    "app will run, but cannot be re-distributed."
-                ),
-                prefix=app.app_name,
+                "Because you are signing with the ad-hoc identity, this "
+                "app will run, but cannot be re-distributed."
             )
```

Comment:
This use of `prefix` looked a bit strange to me since we're not starting a new action or anything....we're just dropped a warning relevant to the immediately preceding log.